contracts/MechMarketplace.sol

```diff
@@ -58,19 +57,25 @@ contract MechMarketplace is IErrorsMarketplace {
     // Maximum response time
     uint256 public immutable maxResponseTimeout;
     // Mech karma contract address
-    address public immutable karmaProxy;
```

Comment:
why changed the name?

Reply:
Cleaner - nobody cares in the contract if it's proxy or not, the setup matters